src/deflate/mod.rs

```diff
@@ -49,9 +49,9 @@ impl Display for Deflaters {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::Libdeflater { compression } => Display::fmt(compression, f),
+            Self::Libdeflater { compression } => Display::fmt(&format!("zc = {}", compression), f),
```

Comment:
I find using the following alternative syntax for the same thing more concise and much easier to read:

```suggestion
            Self::Libdeflater { compression } => write!(f, "zc = {compression}"),
```